client/src/Components/Create/Create.js

```diff
@@ -1,12 +1,122 @@
 
 class Create extends Component {
+  componentDidMount() {
+    $('#location-field').locationpicker({
+      radius: 300,
+      location: {
+        latitude: 47.6062,
+        longitude: -122.3321
+      },
+      inputBinding: {
+        radiusInput: $('[name=radius]')
+      }
+    });
+  }
+
   render() {
     return (
       <div>
-        <h1>Create</h1>
-        <div>This is a test.</div>
+        <div className="container">
+          <div className="row">
+            {/*creation of new meeting title*/}
+            <div className="col-sm-3"></div>
+            <div className="col-sm-6">
+              <h1 id="title">Create a Meeting</h1>
+            </div>
+            <div className="col-sm-3"></div>
+          </div>
+
+          {/*creation of new meeting form*/}
+          <div className="row" id="response-row">
+            <div className="col-sm-3"></div>
+            <div className="col-sm-6">
+              <form id="creation-form" onSubmit={this.submit}>
+                {/*title field*/}
+                <div className="form-group">
+                  <label>Meeting Name</label>
+                  <input name="title" className="form-control" type="text" placeholder="Type the meeting's name" required />
+                </div>
+                {/*closeout time field*/}
+                <div className="form-group">
+                  <label>Close-out Time</label>
+                  <input name="closeoutTime" className="form-control" type="text" placeholder="Type the time the meeting should lock in" required />
+                </div>
+
+                {/*radius field*/}
+                <div className="form-group">
+                  <label>Radius</label>
+                  <input name="radius" className="form-control" type="number" placeholder="Type the radius in meters the meeting should be within" required />
+                </div>
+
+                <div id="location-field" style={{height: '400px'}}></div>
+
+                {/*duration field*/}
+                <div className="form-group">
+                  <label>Duration</label>
+                  <input name="duration" className="form-control" type="number" step="30" min="0" max="600" placeholder="Type the duration of the meeting in minutes" required />
+                </div>
+                {/*name field*/}
+                <div className="form-group">
+                  <label>Your Name</label>
+                  <input name="name" className="form-control" type="text" placeholder="Type your name" required />
+                </div>
+
+                {/*email field*/}
+                <div className="form-group">
+                  <label>Email</label>
+                  <input name="email" className="form-control" type="text" placeholder="Type your email" required />
+                </div>
+
+                {/*location field*/}
+                <div className="form-group">
+                  <label>Location</label>
```

Comment:
The location subview has this title generated already so we can take this out here